### ISSUE TITLE
Floating Label is not centered in the container

### ISSUE THREAD

**Issue by Diego-Franco** [Cnilton/react-native-floating-label-input#27]
Looks like the Floating label is not center either if We pass styles or we use the default values.

As we can see in the "login" input is still not centered

![Screen Shot 2020-12-08 at 4 48 09 PM](https://user-images.githubusercontent.com/5623933/101559161-6bf35c80-3975-11eb-92d6-276811b04857.png)

Code for the previous image:
```
<FloatingLabelInput
	placeholder="Enter a fruit name"
	showErrorMessage={messages.showError}
	value={value}
	onChangeText={onChangeText}
        containerStyles={containerStyle}
	labelStyles={styles.labelStyle}
	inputStyles={styles.inputStyle}
	customLabelStyles={CUSTOM_LABEL_STYLE} />
<FloatingLabelInput
	label="Login"
	value={value}
	onChangeText={nValue => setValue(nValue)}
	customLabelStyles={CUSTOM_LABEL_STYLE}/>
```

Styles code: 
```
const CUSTOM_LABEL_STYLE = {
	fontSizeFocused: 12,
	fontSizeBlurred: 16,
	colorBlurred: 'grey',
	colorFocused: 'grey',
	leftFocused: 0,
	leftBlurred: 0,
	topFocused: 0,
	topBlurred: 0,
};
container: {
		height: 64,
		borderWidth: 2,
		borderRadius: 8,
		alignContent: 'center',
		paddingVertical: 8,
		paddingHorizontal: 16
	},
labelStyle: {
		left: 0, 
		marginLeft: 8
	},
inputStyle: {
		color: 'black',
	},
```

**Comment by Cnilton** [comment]
Hi @Diego-Franco, can you please check and test the new version 1.2.13?

**Comment by Diego-Franco** [comment]
this is fixed in 1.2.13, thank you @Cnilton
